Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 0406d8a299037a9db48ddda2b274546811fb3130)
+++ b/.gitignore	(revision b1ee8e26df44305939cfc291d141bc155ff82fd6)
@@ -21,7 +21,7 @@
 .settings/
 *.sublime-workspace
 .angular
-
+vite
 # Visual Studio Code
 .vscode
 .vscode/*
